android/app/src/main/java/app/organicmaps/car/util/LocationUtils.java

```diff
@@ -0,0 +1,19 @@
+package app.organicmaps.car.util;
```

Comment:
This helper is not car-specific. Moreover, car & navigation should always require FINE location. It doesn't make sense to navigate using data from COARSE location.